ios/SafeAreaView/RNCSafeAreaProvider.m

```diff
@@ -0,0 +1,67 @@
+// Simplified version of https://github.com/facebook/react-native/blob/master/React/Views/RNCSafeAreaView/RCTRNCSafeAreaView.m
```

Comment:
I think it was there before but can you remove this comment, it is not really relevent anymore.